service/lxd.go

```diff
@@ -721,7 +754,7 @@ func (s *LXDService) Restart(ctx context.Context, timeoutSeconds int) error {
 }
 
 // waitReady repeatedly (500ms intervals) asks LXD if it is ready, up to the given timeout.
-func (s *LXDService) waitReady(ctx context.Context, c lxd.InstanceServer, timeoutSeconds int) error {
+func (s *LXDService) waitReady(ctx context.Context, c lxd.InstanceServer, timeoutSeconds time.Duration) error {
```

Comment:
IIRC we talked about this last week, can't the timeout be set through the provided context from the outside?

Reply:
Fixed.